frontend/src/components/common/Grid/components/HeaderCell/style.ts

```diff
@@ -0,0 +1,28 @@
+import styled from "@emotion/styled";
+import { fontBodyXxs, getColors } from "czifui";
+
+export const HeaderCell = styled.span`
+  display: flex;
+  gap: 4px;
+
+  span {
+    min-width: 0; /* facilitates breaking of word on columns; flex default for min width is "auto" */
+  }
+`;
+
+export const CountAndTotal = styled.span`
+  ${fontBodyXxs}
+  ${(props) => {
+    const colors = getColors(props);
+    return `
+      background-color: ${colors?.gray[200]};
+    `;
+  }}
+  border-radius: 20px;
+  color: #000000;
+  display: flex;
+  flex: none;
```

Comment:
Didn't know this is equivalent to `flex: 0 0 auto` v cool! 🎉 

https://stackoverflow.com/a/41773340/3120863